Description: Remove broken test
 This test is failing only in Python 3.5 and 3.6.
 .
 This test is failing, while in fact the framework is doing exactly what
 it should: raise an error. So the test is obviously wrong.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: https://github.com/pecan/pecan/issues/94
Last-Update: 2018-08-24

Index: python-pecan/pecan/tests/test_conf.py
===================================================================
--- python-pecan.orig/pecan/tests/test_conf.py
+++ python-pecan/pecan/tests/test_conf.py
@@ -158,22 +158,6 @@ class TestConf(PecanTestCase):
                 f.name
             )
 
-    def test_config_with_non_package_relative_import(self):
-        from pecan import configuration
-        with tempfile.NamedTemporaryFile('wb', suffix='.py') as f:
-            f.write(b_('\n'.join(['from . import variables'])))
-            f.flush()
-            configuration.Config({})
-
-            try:
-                configuration.conf_from_file(f.name)
-            except (ValueError, SystemError, ImportError) as e:
-                assert 'relative import' in str(e)
-            else:
-                raise AssertionError(
-                    "A relative import-related error should have been raised"
-                )
-
     def test_config_with_bad_import(self):
         from pecan import configuration
         path = ('bad', 'importerror.py')
